modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -8,7 +8,7 @@
 
 You need to create the `install-config.yaml` and the `agent-config.yaml` configuration files so that you can use the Agent-based Installer to generate a bootable ISO image. The Agent-based installation comprises a bootable ISO that contains the Assisted discovery agent and the Assisted Service. Both of these components are required to perform the cluster installation, but the latter component runs on only one of the hosts.
 
-In a subsequent procedure, you can upload your generated Agent ISO image to Oracle’s default Object Storage bucket, which is the initial step for integrating your {product-title} cluster on {oci-first}.
+In a subsequent procedure, you can upload your generated discovery ISO image to Oracle’s default Object Storage bucket, which is the initial step for integrating your {product-title} cluster on {oci-first}.
```

Comment:
for the agent installer, we do not call it a `discovery` ISO. Its a terminology used by the assisted installer. Please just say `generated ISO`